src/opts.rs

```diff
@@ -48,6 +48,41 @@ pub struct MoveOptions {
     pub dump_rebase_plan: bool,
 }
 
+/// Options for rebasing commits. These are the same as MoveOptions, but without the shorthand -m.
+#[derive(Args, Debug)]
+pub struct RebaseOptions {
```

Comment:
OK, that makes sense. To confirm, you mean that we don't need any of these as options via CLI flags on `git reword`, and that instead we can just use/set these default values directly, during the rebase build/execution?

Reply:
@claytonrcarter yep, that's right, you would manually construct a `MoveOptions` value and pass it in where necessary. (I guess it's misleading that one thing is called "move options" and another thing is called a "rebase plan", instead of both of them being called either "move" or "rebase".)